clonelists/Sony - PlayStation Portable (Redump).json

```diff
@@ -249,6 +258,9 @@
 			["Padrino, Il", 1],
 			["Pate, Der", 1]
 		],
+		"Gods Eater Burst": [
```

Comment:
Because these are effectively special editions, the following might be more appropriate:

```
"God Eater": [
			["Gods Eater Burst", 0],
			["God Eater Burst", 0]
		],
```

Reply:
In fact, Gods Eater (the first edition of the game) was released exclusively in Japan with added content through "God Eater Burst - Append-ban". Later, God Eater: Burst was released worldwide with GE+Burst content, so I though that Gods Eater and the Append-ban were useless.